bodhi/server/consumers/masher.py

```diff
@@ -902,6 +903,36 @@ def compose_atomic_trees(self):
         mashed_repos = dict([('-'.join(os.path.basename(repo).split('-')[:-1]), repo)
                              for repo in self.state['completed_repos']])
         for tag, mash_path in mashed_repos.items():
+            #  TODO: Find out the actual values for the tag
+            #  also for the code below, should we use the mash_path instead
+            #  of current_workdir?
+            #  * Pungi config repos hsa $arch, does it matter in bodhi?
+            #  * Do we need any other config files in the pungi-fedora config directory before we fireup the pungi
```

Comment:
If you go the `pungi-koji` way, then you should probably clone everything from the repo. The config files can include other files, so it's hard to tell what exactly you need. Admittedly this is not used currently for Fedora, but it would be nice to be future proof.

If you call just the `pungi-make-ostree` command, you would need some config from https://pagure.io/fedora-atomic (particularly the fedora-atomic-docker-host.json file) for ostree and no special files for Pungi itself.